catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/TemplateTransformer.java

```diff
@@ -56,13 +56,21 @@
 public class TemplateTransformer {
   private static final Logger LOGGER = LoggerFactory.getLogger(TemplateTransformer.class);
 
+  private final FilterReader reader;
```

Comment:
Moving the factory around in the other file makes it safe to do this - and now the same JAXBContext gets used on every request.